projects/ondofinance/index.js

```diff
@@ -19,9 +20,18 @@ const config = {
   },
   mantle: {
     USDY: "0x5bE26527e817998A7206475496fDE1E68957c5A6",
+  },
+  sui: {
+    USDY: "0x960b531667636f39e85867775f52f6b1f220a058c4de786905bdf761e06a56bb::usdy::USDY"
```

Comment:
you can ignore the error, but what is the decimals fields for this, I think we need to manually add mapping in our server for this

Reply:
USDY on SUI is 6 decimals, thanks!
https://suiscan.xyz/mainnet/coin/0x960b531667636f39e85867775f52f6b1f220a058c4de786905bdf761e06a56bb::usdy::USDY/txs